### ISSUE TITLE
Loss function drmsd component commented out

### ISSUE THREAD

**Issue by TrentBrick** [OpenProtein/openprotein#16]
I was wondering why you commented out the drmsd component of the loss function while training?

**Comment by JeppeHallgren** [comment]
What line are you referring to here?

**Comment by TrentBrick** [comment]
Lines 53 and 56 of `openprotein.py`

**Comment by TrentBrick** [comment]
While on the topic of loss functions, am I right that you apply the mask to the tertiary structure **and** primary sequence when you preprocess the data rather than masking it in the loss function? 

This seems bad to me because you are changing the protein itself before it is presented to the neural network and removing potential interactions the model should be acknowledging in making its angle predictions.

**Comment by JeppeHallgren** [comment]
Hey @TrentBrick ,
My assumption was that most people would actually want to define their own loss function and overwrite `def compute_loss` in `openprotein.py`. Happy to discuss what default loss function makes the most sense - I quickly found that drmsd can have some very steep gradients (changing just one dihedral angle can cause the global drmsd loss to explode).

**Comment by JeppeHallgren** [comment]
Processing the masks in the preprocessing was purely done for efficiency reasons (faster that doing it in every training iteration). But of course if a given model needs the masked values, the preprocessing script should have an option not to apply the mask.
